### ISSUE TITLE
Allow logos to be stored for each object

### ISSUE THREAD

**Issue by wrussell1999** [MLH-Fellowship/orientation-project-python-23.FAL#2]
All Experience, Education and Skill objects contain a logo attribute. Figure out a way to store these images with user friendliness in mind.

The client should identify if the logo field is empty and display this field.

**Comment by boushrabettir** [comment]
I can work on this!

**Comment by boushrabettir** [comment]
**Question:** Are the logos planned to be stored as type `string`?

**Comment by jescalada** [comment]
I think it would be good to store the logos as URLs. That being said, I'm not sure what the details would be. Should we be storing them in the server itself (as a BLOB)? Or just simplify and store as strings? 🤔

**Comment by boushrabettir** [comment]
> I think it would be good to store the logos as URLs. That being said, I'm not sure what the details would be. Should we be storing them in the server itself (as a BLOB)? Or just simplify and store as strings? 🤔

Yeah, I wish we had some more details about this. I am meeting with our pod leader tomorrow to discuss details about this.
